src/code.sh

```
@@ -81,150 +53,359 @@ _trim_fastq_endings () {
+
+       # run FusionInspector, adding an arg to run Trinity if requested by user, and adding optional user-entered parameters if any 
+       if [ "$include_trinity" = "true" ]; then
+              mark-section "Adding Trinity de novo reconstruction option to FusionInspector command"
+              fusion_ins="${fusion_ins} --include_Trinity"
+       fi
+
+       if [ -n "$opt_parameters" ]; then
+              # Test that there are no banned parameters in --parameters input string
+              banned_parameters=(--fusions -O --CPU --left_fq --right_fq --out_prefix --genome_lib_dir --vis \
+              --examine_coding_effect --extract_fusion_reads_file --include_Trinity)
+              for parameter in ${banned_parameters[@]}; do
```

Shellcheck (0.10.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)